forui/test/src/theme/theme_test.dart

```diff
@@ -21,11 +21,9 @@ class Foo extends StatelessWidget {
 void main() {
   group('FTheme', () {
     testWidgets('ThemeData is visible in child widgets', (tester) async {
-      await tester.pumpWidget(
-        Foo(
-          child: Builder(builder: (context) => Text('${context.theme == FThemes.zinc.dark}')),
-        ),
-      );
+      await tester.pumpWidget(Foo(
+        child: Builder(builder: (context) => Text('${context.theme == FThemes.zinc.dark}')),
+      ));
```

Comment:
Trailing comma. I think we should enable `require_trailing_commas` lint.

https://dart.dev/tools/linter-rules/require_trailing_commas

https://github.com/forus-labs/cauldron/blob/6d03bffba7cb83919cfcf01dca8ea5c3b47b3c12/flint/lib/analysis_options.dart.yaml#L207

Reply:
I'll enable it in flint over the next few days.